src/main/kotlin/org/jitsi/jibri/service/impl/SipGatewayJibriService.kt

```diff
@@ -85,14 +88,24 @@ class SipGatewayJibriService(
     override fun start() {
         jibriSelenium.joinCall(
             sipGatewayServiceParams.callParams.callUrlInfo.copy(urlParams = SIP_GW_URL_OPTIONS))
-        whenever(jibriSelenium).transitionsTo(ComponentState.Running) {
+
+        // when in auto-answer mode we want to start as quick as possible as
+        // we will be waiting for a sip call to come
+        if (sipGatewayServiceParams.sipClientParams.autoAnswer) {
             pjsuaClient.start()
+        } else {
+            whenever(jibriSelenium).transitionsTo(ComponentState.Running) {
+                pjsuaClient.start()
+            }
         }
     }
 
     override fun stop() {
         processMonitorTask?.cancel(false)
-        pjsuaClient.stop()
-        jibriSelenium.leaveCallAndQuitBrowser()
+        try {
```

Comment:
this try/catch should no longer be needed, right?

Reply:
Well, I think for safety it should be there. 
I experienced one issue where the chrome stays opened when kill fails for some reason. 
We know how bad this can be from the recent experience, so I was thinking we better keep the try-finally code :)

Reply:
if that call is throwing, i'd rather we make sure it's handled appropriately there (along with the other exceptions) so that if it's called from elsewhere we don't run into the same problem.

Reply:
if we do want an exception catch-all, we should put it in `JibriSubprocess`

Reply:
But there is no catch-all here. It is just try-finally, just to be sure that whatever happens quitting selenium will be called.

Reply:
sorry, what i mean is that the call to stop shouldn't throw, as it isn't the error-propagation model being used here.  it was a mistake on my part not to be more thorough when checking what exceptions were possible in `JibriSubprocess#stop`, but here we are.  i don't want to put a finally here, as it suggests this code should potentially handle an exception when it shouldn't.  let's remove this part and i will take another pass at the calls in `JibriSubprocess#stop` to make sure all exceptions are handled correctly.

Reply:
Ok, I dropped that commit.

Reply:
thanks, merged